onnxruntime/test/providers/internal_testing/internal_testing_tests.cc

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#if !defined(REDUCED_OPS_BUILD)  // may not work with excluded op kernel implementations
+
```

Comment:
Does the internal testing EP use real kernels? It was originally created to test partitioning and to be a fake compiling EP so I wouldn't have expected reducing the statically registered kernels to affect it.

Reply:
I think there is some part of the tests that relies on falling back to CPU kernels and those may not be present depending on the reduced set of ops.